README.md

```diff
@@ -27,15 +27,17 @@ Both generation pipelines are still in `a work in progress` state, and several i
 
 ## Generated models 
 
-| `YARP_ROBOT_NAME` | Pipeline     | Notes                 |
-|:-----------------:|:------------:|:---------------------:|
-| `iCubDarmstadt01` | simmechanics | v2.5 without backpack |
-| `iCubGazeboV2_5`  | simmechanics | v2.5 with backpack, and inertias of some links modified to run smoothly in Gazebo | 
-| `iCubGenova01`    | simmechanics | v2.5 without backpack | 
-| `iCubGenova02`    | simmechanics | v2.5   with backpack  | 
-| `iCubGenova03`    | dh | v2 with legs v1 and feet v2.5   | 
-| `iCubGenova04`    | simmechanics | v2.5   with backpack  | 
-| `iCubLisboa01`    | dh           | v1 with head v2       |
-| `iCubNancy01`     | dh           | v2.5 with arms v1 and head v2|
-| `iCubParis01`     | dh           | v1 with feet v2.5     | 
-| `iCubParis02`     | dh           | v2 with feet v2.5     | 
+|  `YARP_ROBOT_NAME`   | Pipeline     | Notes                           |
+|:--------------------:|:------------:|:-------------------------------:|
+| `iCubDarmstadt01`    | simmechanics | v2.5 without backpack           |
+| `iCubGazeboV2_5`     | simmechanics | v2.5 with backpack, and inertias of some links modified to run smoothly in Gazebo | 
+| `iCubGazeboV2_5_plus`| simmechanics | v2.5+ with backpack, and inertias of some links modified to run smoothly in Gazebo | 
+| `iCubGenova01`       | simmechanics | v2.5 without backpack           | 
+| `iCubGenova02`       | simmechanics | v2.5   with backpack            | 
+| `iCubGenova02_plus`  | simmechanics | v2.5+   with backpack           |
```

Comment:
Given that "plus" variant has been implemented only on iCubGenova04, would it make sense to have `iCubGenova04_plus` instead of `iCubGenova02_plus`?

Reply:
Correct. Thanks!